src/angular-app/languageforge/lexicon/editor/editor.component.ts

```diff
@@ -128,6 +148,8 @@ export class LexiconEditorController implements angular.IController {
       if (this.hasUnsavedChanges()) {
         this.saveCurrentEntry();
       }
+      // destroy listeners when leaving editor page
```

Comment:
I still don't think this will fire if someone is just navigating around...unless this event is handled specially by Angular, it would only apply for a full page refresh, not a location change in a SPA.

Reply:
Pull it down and add a brakepoint there. It does actually give the behavior I was intending.

Reply:
ok, good enough for me.  I'll approve.  Thanks.